api/job/daemon_test.go

```diff
@@ -9,13 +9,13 @@ import (
 	"github.com/quintilesims/layer0/common/models"
 )
 
-func TestJanitor(t *testing.T) {
+func TestDaemonFN(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 
 	expiry := time.Hour * 12
 	mockJobStore := mock_job.NewMockStore(ctrl)
-	janitor := NewJanitor(mockJobStore, expiry)
+	deamonFN := NewDaemonFN(mockJobStore, expiry)
```

Comment:
[typo] daemonFN

Reply:
I get the feeling I'm gonna make this typo a lot